### ISSUE TITLE
✍️ Contribution period: Azhar Ismagulova

### ISSUE THREAD

**Issue by azharcodeit** [ersilia-os/ersilia#991]
### Week 1 - Get to know the community

- [X] Join the communication channels
- [X] Open a GitHub issue (this one!)
- [X] Install the Ersilia Model Hub and test the simplest model
- [X] Install Docker if needed, and test another model
- [X] Write a motivation statement to work at Ersilia
- [x] Submit your first contribution to the Outreachy site

### Week 2 - Get Familiar with Machine Learning for Chemistry

- [ ] Select a model from the list suggested in GitBook
- [ ] Download and serve the model via the Ersilia Model Hub to ensure it works
- [ ] Open a repository on your GitHub user with all the necessary files
- [ ] Select and clean a dataset of 1000 molecules (example notebook 1)
- [ ] Run predictions for the molecules on the selected model and evaluate the results

### Week 3 - Validate a Model in the Wild

- [ ] Find a suitable dataset with sufficient experimental results
- [ ] Clean and standardize the dataset
- [ ] Run predictions and calculate metrics.

### Week 4 - Prepare your final application

- [ ] Submit the final application in the Outreachy website

**Comment by azharcodeit** [comment]
I am thrilled with the opportunity Ersilia provides regarding data science. In my home country, there are very limited projects to showcase and make use of machine learning models. Back in my first place of work as a data analyst, I could only imagine working with prediction models and felt a bit frustrated about the very small amount of scope of exposure I worked with on this amazing ever-evolving topic as data science. This limitation often resulted in a lack of vacancies in this area.

However, Ersilia provides the opportunity to gain hands-on knowledge and a unique opportunity to collaborate with scientists and people working collaboratively. This exposure reignited my passion for data science and programming with big data.

During my studies in computer science, I pivoted more towards the data science part of the subject. I also worked on writing research papers in the Machine Learning field, further solidifying my interest and expertise in this domain.

Moreover, despite being aware of the challenges faced by women in AI/ML professions, such as the statistic that only 26% of professionals are women, I remain determined to compete and contribute meaningfully. However, there are complexities in securing jobs as a woman, which led me to make pivots like becoming a frontend engineer. Even after relocation, I encountered challenges in finding suitable job opportunities.

The values and aims of Ersilia are truly inspiring. The objective to build ready-to-use AI/ML models and foster an egalitarian approach, making everyone included in receiving treatments, resonates deeply with me. Being part of such an environment would not only be impactful but also propel me to the next level of making an impact.

I deeply appreciate the opportunity to be among scientists and have a PhD-level mentor. This experience helps build a strong foundation, and the compassionate values upheld by the community further enhance the learning environment.

In conclusion, the Ersilia Model Hub internship represents a significant opportunity for me to further my passion for data science and contribute to transformative projects.

**Comment by GemmaTuron** [comment]
Hi @azharcodeit 

We are on week 3 of the contribution period. Please let us know if you plan to continue with your contribution by the next 2 days, otherwise we will close this issue so we can focus on the applicants who want to make a final application to Ersilia.

**Comment by GemmaTuron** [comment]
Hi @azharcodeit 
We wull close this issue to allow mentors to focus their support on applicants who wish to make a final application to Ersilia. Let us know if you were working offline.

Thanks!
